application/modules/front/views/layouts/searchbox.php

```diff
@@ -10,7 +10,7 @@
     <a class="searchtipslink">Search Tips</a>
     <div id="searchbar">
         <form name="searchform" action="/search/" method=get id="searchform">
-            <input type="text" class="searchquery" name=q placeholder= "Search …" value="<?php echo htmlspecialchars(strip_tags($stextval)); ?>" />
+            <input type="text" class="searchquery" name=q placeholder= "Search..." value="<?php echo htmlspecialchars(strip_tags($stextval)); ?>" />
```

Comment:
I prefer the ellipsis symbol lol.

Reply:
The reason for doing this was to increase clarity and visibility. I thought the symbol was a little less visible.

However, this is an old version of the code, I changed it once again in a later commit... (#ellipses: pun intended)

Reply:
This is what I was saying before that this UI update was not a finish-one-thing-cleanly-and-completely, then move onto another thing. Inspiration struck me at different points in time. I changed one thing, moved onto another thing, I liked that, committed them both at some point, then while I was passing by the previous thing, I got another idea, and updated the first thing, or maybe because I liked the second thing more, I wanted the first thing to be more consistent or closer to the looks of the second thing. I guess creativity works like that—it can come unannounced at any point in time. 
If I was being too strict with my planning and not touching anything afterwards once it was done, I would have lost a lot of the good inspiration and consistency as the design evolved over time.
UI design is a little different from a planned functionality implementation for which you know you want to be able to specifically do a certain thing, and it is known beforehand. In UI design, you don't completely know what you want at the beginning, you just have a vague idea, and you build it by looking and feeling it, and it evolves over time.

Reply:
Yes, I understand. I'm reviewing in a different way now, looking at the final changes instead of individual commits.